dockerfiles/Dockerfile-delegation-backend

```diff
@@ -5,7 +5,7 @@ WORKDIR $GOPATH/src/delegation_backend
 
 # Copy everything from the current directory to the PWD (Present Working Directory) inside the container
 COPY src src
-COPY result/headers src/delegation_backend
```

Comment:
Is this change intentional? Doesn't seem that this PR changes anything in structure

Was build via Docker tested?

Reply:
Previously the code structure has changed so that has to be reflected in the Dockerfile aswell. It was tested, I will include it in a sepparate commit.